extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -400,6 +406,16 @@ export class AmpAutocomplete extends AMP.BaseElement {
           }
           child.classList.add('i-amphtml-autocomplete-item');
           child.setAttribute('role', 'listitem');
+          const valueToReplace = child.getAttribute('data-value');
```

Comment:
Let's also discuss this before merging, but I think in past discussions we wanted this feature to be configurable